virttest/test_setup.py

```diff
@@ -578,6 +584,17 @@ def get_multi_supported_hugepage_size(self):
             raise ValueError("Root hugepage control sysfs directory %s did not"
                              " exist" % self.pool_path)
 
+    def get_hugepages_surp(self):
```

Comment:
we can actually reuse `read_from_meminfo('HugePages_Surp')` in `virttest/staging/utils_memory.py`

Reply:
Good point, will update it with other suggestions